src/Gold.Redis/Gold.Redis.Common/Interfaces/Db/IRedisKeysDb.cs

```diff
@@ -0,0 +1,9 @@
+﻿using System.Threading.Tasks;
+
+namespace Gold.Redis.Common.Interfaces.Db
+{
+    public interface IRedisKeysDb : IRedisDbGeneralOperations
+    {
+        Task<bool> Set<T>(string key, T value);
```

Comment:
please implement set command before pushing to develop. it's pretty standard command and should really be part of this branch